### ISSUE TITLE
Modify the pages to be responsive on mobile

### ISSUE THREAD

**Issue by AlexandreBonneau** [autoNumeric/autonumeric.github.io#2]
The landing page is awful on small screen.
Fix that.

**Comment by MatthewElson** [comment]
Is this project still active?

**Comment by AlexandreBonneau** [comment]
As you can see by the response time, yes it is, in a way :)

However since I cannot approach the speed of light to slow time (and somehow still stay here on Earth !), I haven't found the time to tackle this issue.

Any help is welcome though! 😎

**Comment by MatthewElson** [comment]
What you can't approach the speed of light? I thought everybody could do that... maybe it's just me 😃 

I'm trying to get the landing page up and I'm having some difficulties, what's the project built with?
Anything else I need to know would be helpful!

**Comment by AlexandreBonneau** [comment]
Ah!
The website *source* code is in the [autonumeric.org](https://github.com/autoNumeric/autonumeric.org) repository.
This `autonumeric.github.io` repository is only used for the *compiled* code, served automatically by the github pages.

The underlying technologies for the website are Vue.js and the Quasar Framework.
You can check the readme on the repo for build instructions, it's [pretty straightforward](https://github.com/autoNumeric/autonumeric.org#how-to-contribute)!

**Comment by MatthewElson** [comment]
Ok, making progress... however I can't get the page to load properly. This is what I get when I do "yarn dev" 
![image](https://user-images.githubusercontent.com/32372004/33728453-6fbe65e6-db48-11e7-8304-d9123f4a14bc.png)
By chance do you know how to fix this?

**Comment by AlexandreBonneau** [comment]
Yes I do; the error message is pretty explicit though : `Expected linebreaks to be 'LF' but found 'CRLF'`. :)

This means the code editor your are using is **very** unpolite and uses the windows-only 'new line' characters (`\r\n`), instead of respecting the one already setup in the files (`\n`).

Luckily, `eslint` caught that error and warned you; it would have been a pain to see which line your commit changed if *all* the lines are modified by that editor.

I would suggest using one of those smart code editors (in that order):
- PhpStorm
- VSCode
- Sublime
- Kate

**Comment by MatthewElson** [comment]
So that's what I read online too. I just wasn't sure how to fix it. Although I don't understand how this is the code editors fault. I haven't change any code or anything at this point.

**Comment by AlexandreBonneau** [comment]
Some editors just have a default newline character setup, and just update all of those on save.
If you see such editor, report it to the police though ;)

**Comment by MatthewElson** [comment]
Right, however I haven't opened a text editor or saved/changed any files yet either. This is happening in the initial build. 

What I've done so far:
Fork the repository
Open GIT Bash
cd /where my github files go
Clone repository to my machine
cd /where my github files go/autonumeric.org
(yarn's already installed so I skipped that step) 
yarn - it gets all the dependencies and packages correctly
yarn dev - which is when I get the error 

Just so you know I'm trying to figure this out myself too. I'm not just bugging you for support. XD

**Comment by AlexandreBonneau** [comment]
It's ok :)

Well, I stopped using Windows so long ago, I can only make conjectures about what has gone wrong here unfortunately.

Have you tried within WSL?

**Comment by MatthewElson** [comment]
Ok, so I got it... I opened the file associated with the error in Sublime Text 2 then went view > line endings > Unix for each one then saved the file. I feel like there's probably an easier way than that but I only had to do it for about 10 files. Now I can finally get started! Thanks for the help!

**Comment by MatthewElson** [comment]
Do you want me to create a PR here or create an issue and PR at autonumeric.org instead?

**Comment by AlexandreBonneau** [comment]
It would be preferable on the autonumeric.org repository indeed.

Just link the new issue to this one then.

Also, thanks ;)
